includes/class.llms.person.handler.php

```diff
@@ -676,6 +676,9 @@ public static function login( $data ) {
 		// Validate the fields & allow custom validation to occur.
 		$valid = self::validate_fields( apply_filters( 'lifterlms_user_login_data', $data ), 'login' );
 
+		// Allow for custom validation of user login data (previous filter doesn't fully allow for this)
+		$valid = apply_filters( 'llms_after_user_login_data_validation', $valid, $data, 'login' );
```

Comment:
Docblocks & dropped the 3rd parameter that isn't really necessary (though it does conform with other similar filters)

```suggestion
		/**
		 * Filters the validation result of user-submitted login data
		 * 
		 * @since [version]
		 *
		 * @param WP_Error|boolean $valid An error object containing validation errors or `true` if no validation errors found.
		 * @param array            $data  User submitted login data.
		 */
		$valid = apply_filters( 'llms_after_user_login_data_validation', $valid, $data );
```